docs/accessing_transaction_field.rst

```diff
@@ -7,43 +7,60 @@ PyTeal smart contracts can access properties of the current transaction and the
 blockchain when they are running.
 
 Transaction Fields
-------------------
+##################
+
+Information about the current transaction being evaluated can be obtained using the :any:`Txn` object using the PyTeal expressions shown below.
+
+Since numerous transaction fields are available, the fields are logically organized into tables matching the structure of `go-algorand's Transaction struct definition <https://github.com/search?q=%22type+Transaction+struct%22+repo%3Aalgorand%2Fgo-algorand+path%3A%2Fdata%2Ftransactions&type=Code>`_.
```

Comment:
@jasonpaulos I intentionally linked via search in case the definition moves.  I generally either link by commit or search.  In my experience, linking to an active branch (e.g. `master`) with a line number is prone to becoming stale.

Nevertheless, since you felt the link is unnecessary, I removed it in https://github.com/algorand/pyteal/pull/204/commits/86a036781943cbb3155ba24740c1b3f24d524437.

Reply:
Thanks. Though for future reference, by permalink I mean a link to a specific commit. For example:
* Normal link: `https://github.com/algorand/pyteal/blob/master/pyteal/ast/txn.py#L274`
* Permalink: `https://github.com/algorand/pyteal/blob/9521f62e9c0f442fe4bc3890c3cc1791704dd42e/pyteal/ast/txn.py#L274` (obtained by using the "Copy permalink" button on GitHub from the "normal link")

Reply:
@jasonpaulos Gotcha - Noted on preference for commit link over search link.  

I take your reply to mean thread is resolved.  Resolving.